test_plans/spark-processing

```diff
@@ -0,0 +1,21 @@
+bundle: bundle:spark-processing
+benchmark:
+    spark:
+        logisticregression
+    spark:
+        matrixfactorization
+    spark:
+        pagerank
+    spark:
+        sql
+    spark:
+        streaming
+    spark:
+        svdplusplus
+    spark:
+        svm
+    spark:
+        trianglecount
+bundle_name: spark-processing
+bundle_file: bundle.yaml
+url: https://jujucharms.com/spark-processing/
```

Comment:
This doesn't break anything. In near future, would like to add a link back to the charm store.